_posts/2023-03-14-nome-in-codice-italia.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+lang: it
+excerpt_separator: <!-- MORE -->
+title: "Nome in codice: ___ Italia"
+subtitle: "Partecipa al sondaggio per definire il nuovo nome del design system del Paese"
+date: 2023-03-14 09:00:00 +0200
+pinned: false
+nodate: false
+homeBottomLink: false
+homeHideCategories: true
+date_prepend: "Articolo"
+categories:
+  - cittadini
+  - pubblica amministrazione
+---
+
+<!-- MORE -->
+**Designers Italia lancia un sondaggio per definire il nome del design system**, una delle risorse fondamentali per realizzare progetti digitali della Pubblica Amministrazione di qualità, accessibili, e inclusivi. La consultazione informale sarà aperta per un mese su Forum Italia, fino al **14 aprile.**
+
+## Cos’è il design system
+Il [design system](https://prossima.designers.italia.it/design-system/) può essere visto come una **"scatola dei mattoncini"** che contiene, al suo interno, una serie di librerie di componenti pronti all'uso, principi e guide pratiche per progettare interfacce digitali, insieme a una raccolta di soluzioni pronte all’uso (design pattern) e casi studio. Una serie di strumenti a disposizione di chi progetta e sviluppa interfacce digitali che permette a PA, in-house e fornitori di **progettare esperienze semplici e che funzionano**, risparmiare tempo e risorse, oltre che di usufruire di materiali open source costantemente aggiornati, documentazione tecnica, esempi e buone pratiche di versionamento. 
```

Comment:
@gianvitofanelli in questa frase c'è due volte "all'uso", sia componenti che design pattern, è corretto?